lib/src/client/connection.dart

```diff
@@ -14,17 +14,19 @@
 // limitations under the License.
 
 import 'dart:async';
-import 'dart:convert';
-import 'dart:io';
 
-import 'package:http2/transport.dart';
 import 'package:meta/meta.dart';
 
-import '../shared/timeout.dart';
-
+import '../shared/status.dart';
 import 'call.dart';
 import 'options.dart';
 
+import 'transport/http2_transport_stub.dart'
```

Comment:
For me this wasn't buildable using `build_runner`. The following error occurred: 

```
[SEVERE] build_web_compilers|ddc on package:grpc/grpc.dartdevc.module:
Error compiling dartdevc module:grpc|lib/grpc.ddc.js

[error] Target of URI doesn't exist: 'transport/http2_transport.dart'. (package:grpc/src/client/connection.dart, line 25, col 26)
[error] Target of URI doesn't exist: 'transport/xhr_transport_stub.dart'. (package:grpc/src/client/connection.dart, line 27, col 8)

Please fix all errors before compiling (warnings are okay).
}
```

Weirdly I was able to fix this by adding "ignore: uri_does_not_exist". I saw this approach within the `http` package.

```
// ignore: uri_does_not_exist
import 'transport/http2_transport_stub.dart'
    // ignore: uri_does_not_exist
    if (dart.library.io) 'transport/http2_transport.dart';
import 'transport/transport.dart';
// ignore: uri_does_not_exist
import 'transport/xhr_transport_stub.dart'
// ignore: uri_does_not_exist
    if (dart.library.html) 'transport/xhr_transport.dart';
```

Any idea why this is necessary?